src/components/Button/Button.vue

```diff
@@ -13,6 +13,15 @@
         'tertiary bg-white border border-gray-100 text-gray-500 active:border-gray-300 disabled:border-white-300':
           tertiary
       },
+      {
+        'success bg-success !text-white active:bg-success-700': success
+      },
+      {
+        'error bg-error !text-white active:bg-error-700': error
+      },
+      {
+        'bg-opacity-50': (success || error) && disabled
+      },
```

Comment:
@mercedesb what am I missing here, that adding `disabled:bg-opacity-50` or even `disabled:!bg-opacity-50` in the `success` or `error` objects doesn't do what it's supposed to?

Reply:
Ohhh good question! Disabled is a tailwind variant that isn't enabled for all the core plugins by default. You'll have to go to the `tailwind-plugin-lob` repo and add a variant for opacity ([example here for borderColor](https://github.com/lob/tailwind-plugin-lob/blob/main/index.js#L131)) and then Beth will have to publish a new package for you

Reply:
Thanks, opened https://github.com/lob/tailwind-plugin-lob/pull/1